examples/stage/odc_eks/odc_eks.tf

```diff
@@ -1,3 +1,7 @@
+locals {
+  create_certificate = true
```

Comment:
Why define a local and not use a variable like we do with all the others?

Reply:
I needed local var here so I can utilise it in output like - 
```
output "certificate_arn" {
  value = (local.create_certificate)? module.odc_eks.certificate_arn : ""
}
```